Translation corrections (#495)
07th-mod/higurashi-console-arcs: Update/kage_tips_131.txt

//rそれに教官もたいした面子が揃っていないため、志望するのはよほどの物好きか、技量的にいまひとつの一軍半だと聞いていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それに教官もたいした面子が揃っていないため、志望するのはよほどの物好きか、技量的にいまひとつの一軍半だと聞いていた。",

> 			NULL, "Also, since the instructors didn't have a lot name recognition, I heard only those who were really curious or skilled aimed for it, only to get nowhere.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Also, the instructors didn't have a lot name recognition, so I heard only those who were really curious or less skilled aimed for it.", Line_Normal);
```

			NULL, "\"With your skills and achievements, couldn't you get into the Tokyo College of the Arts?", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3090", 256, TRUE);
	OutputLine(NULL, "　今の高校みたいに、奨学金とか特待生待遇とか…」",

> 			NULL, " With all those honors and special scholarships like the ones you've gotten at this school...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " With all those honors and special scholarships like you've gotten at this school...\"", Line_Normal);
```

			NULL, "\"Well now, it sounds like you've really fallen in love.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3094", 256, TRUE);
	OutputLine(NULL, "いくら告白したてで不安だからって、自分の将来まで相手の都合で決めなくてもいいんじゃない？」",

> 			NULL, " I know you've only just confessed and are nervous, but you don't have to let her circumstances decide your future, right?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I know you've only just confessed and you're still nervous, but you don't have to let her circumstances dictate your future, right?\"", Line_Normal);
```

//rそれは、あの子の今の成績だと明らかにきつかったし、私みたく浪人覚悟で受けるほどに家の人も理解はないと聞いていたからなんだけど…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "それは、あの子の今の成績だと明らかにきつかったし、私みたく浪人覚悟で受けるほどに家の人も理解はないと聞いていたからなんだけど…。",

> 			NULL, "It was obviously a poor choice given her current grades. And I'd heard that her family, unlike mine, wouldn't allow her to aim for any universities apart from ones whose entrance exams she could pass on the first try...", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was obviously a poor choice given her current grades. And unlike my family, I heard that her family wouldn't allow her to aim for any universities except the ones where she could pass the entrance exams on the first try...", Line_Normal);
```

			NULL, "\"Yeah. ...There might be a slight issue.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3101", 256, TRUE);
	OutputLine(NULL, "あの子にはそのこと、まだ黙っててくれるかな？」",

> 			NULL, " Could you keep quiet about it to her for now?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Could you keep this quiet from her for now?\"", Line_Normal);
```
